Content.Server/GameObjects/Components/Kitchen/MicrowaveComponent.cs

```diff
@@ -297,6 +301,14 @@ private void wzhzhzh()
                 {
                     return;
                 }
+
+                if(failState == MicrowaveSuccessState.UnwantedForeignObject)
+                {
+                    VaporizeReagents();
+                    EjectSolids();
+                    goto Finish;
```

Comment:
Gotos are considered bad practice in nearly all languages. The rest of this should just be wrapped in an `else`.